pypesto/select/problem.py

```diff
@@ -21,11 +20,10 @@ class Problem:
     Attributes
     ----------
     calibrated_models:
-        Storage for all calibrated models. A dictionary, where keys are
-        model hashes, and values are :class:`petab_select.Model` objects.
+        All calibrated models.
```

Comment:
would it make sense to link the `Models` class here?

Reply:
Yes, done now, thanks